src/client/private-pages/index.html

```diff
@@ -19,6 +20,7 @@
   <app-element>
     <dash-page id="dash"></dash-page>
     <dash-pagination id="all"></dash-pagination>
+    <github-app-config id="github-app-config"></github-app-config>
```

Comment:
`/org/config/:orgname` would be my preference. Redirect on GitHub install from installation id => org config URL.